### ISSUE TITLE
NTR: Joint imaging: loose body in joint (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8973]
New term request Joint imaging: loose body in joint (ang_130721215804): Foreign body, radiodense fragments  UMLS: 0022411

**Comment by pnrobinson** [comment]
@MickeySegal
is this an iatrogenic finding? What is the context?

**Comment by MickeySegal** [comment]
A variety of conditions such as Foreign body arthritis, Pigmented villonodular synovitis, Synovial chondromatosis and Osteochondritis dissecans.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000591
- label: Loose body in joint
- parents: [Abnormal joint morphology (HP:0001367)](https://hpo.jax.org/app/browse/term/HP:0001367)
- pmid: PMID:30451404
- def: Small fragments of articular cartilage that break off in a joint as a result of a injury, degeneration, or other processes.
- comment: This abnormality may be observed upon joint imaging in individuals with diseases such as foreign body arthritis, pigmented villonodular synovitis, synovial chondromatosis, and osteochondritis dissecans.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8973
- synonyms:
